File: Eclipse 2/codeLM starter-code/Python/extraction.py
def extraction(casino_floor):
  # Your code here


if __name__ == '__main__':
  casino_floor_row = int(input())
  casino_floor_col = int(input())
  casino_floor = []
  for _ in range(casino_floor_row):
    casino_floor.append([])
    for _ in range(casino_floor_col):
      casino_floor[-1].append(input())

  function_call_result = extraction(casino_floor)
  for x in function_call_result:
    for y in x:
      print(y, end=' ')
    print()

File: Eclipse 2/codeLM starter-code/Java/Extraction.java
import java.util.Scanner;

class Extraction {
    static int[][] extraction(char[][] casinoFloor) {
        // Your code here
    }

    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);

        int casinoFloorRow = Integer.parseInt(s.nextLine());
        int casinoFloorCol = Integer.parseInt(s.nextLine());
        char[][] casinoFloor = new char[casinoFloorRow][casinoFloorCol];
        for (int i = 0; i < casinoFloorRow; i++) {
            for (int j = 0; j < casinoFloorCol; j++) {
                casinoFloor[i][j] = s.nextLine().charAt(0);
            }
        }

        int[][] functionCallResult = extraction(casinoFloor);
        for (int i = 0; i < functionCallResult.length; i++) {
            for (int j = 0; j < functionCallResult[i].length; j++) {
                System.out.print(functionCallResult[i][j]);
                System.out.print(" ");
            }
            System.out.println();
        }
    }
}